src/compas/geometry/polyhedron.py

```diff
@@ -491,7 +491,7 @@ def from_convex_hull(cls, points):
         --------
         >>> from compas.geometry import Polyhedron
         >>> points = [[0, 0, 0], [1, 0, 0], [0, 1, 0]]
-        >>> p = Polyhedron.from_convex_hull(points)
+        >>> p = Polyhedron.from_convex_hull(points)  # doctest: +SKIP
```

Comment:
It would be nice if we also have a way document the minimal plugins needed for certain functions